electron-component/scripts/dor-le-dor.js

```diff
@@ -0,0 +1,9 @@
+const {ipcRenderer} = require('electron');
+
+
+function sendMessage(event) {
+    //let topStoryText = document.getElementsByClassName("top-story-text")[0];
+    ipcRenderer.send('asynchronous-message', event.data);
+}
+
+window.addEventListener("message", sendMessage, false);
```

Comment:
1. you need to make sure that this message "belongs" to you before you forward it
2. you don't have a reply mechanism, you won't be able to return a message on reply

(the same send+reply mechanism should be used in the dorldor and whatsapp pages)

Reply:
1.done
2.will be done in the future